Typo on parameter

JackStromberg opened the pull request:
Invalid parameter for New-AzureRmNetworkWatcherPacketCapture; Changed -Filters to -Filter.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

JackStromberg committed:
Typo on parameter

Invalid parameter for New-AzureRmNetworkWatcherPacketCapture; Changed -Filters to -Filter.
